src/standalone/Snow/Snow.jl

```diff
@@ -272,8 +368,13 @@ function ClimaLand.make_update_aux(model::SnowModel{FT}) where {FT}
 
         @. p.snow.T_sfc = snow_surface_temperature(p.snow.T)
 
-        @. p.snow.water_runoff =
-            compute_water_runoff(Y.snow.S, p.snow.q_l, p.snow.T, parameters)
+        @. p.snow.water_runoff = compute_water_runoff(
+            Y.snow.S,
+            p.snow.q_l,
+            p.snow.T,
+            p.snow.ρ_snow,
+            parameters,
+        ) #do we want to dispatch this off the density model type so that it can take Y.snow.Z insetad of p.snow.ρ_snow when it exists to avoid a redundant computation?
```

Comment:
what would be better than making this function have multiple methods is to make a helper function:

snow_depth(density_model, Y, p) which returns either Y.snow.Z or computes depth using density, and use this internally to compute_water_runoff. 

Although depth is easy to compute from density, it may be less confusing to do this ^^ than to compute density from Y.snow.z, then compute depth from this...